### PR TITLE
add TUNERCF435 config

### PR DESCRIPTION
**Pull request by TUNERC-Aria**
Samples and schematics were provided on the Discord private channel.

### CODE REVIEW

File: configs/TUNERCF435/config.h [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#define UART1_TX_PIN            PA9
+#define UART2_TX_PIN            PA2
+#define UART3_TX_PIN            PB11
+#define UART4_TX_PIN            PA0
+#define UART1_RX_PIN            PA10
+#define UART2_RX_PIN            PA3
+#define UART3_RX_PIN            PB10
+#define UART4_RX_PIN            PA1
+
+#define INVERTER_PIN_UART4		PC15
```
Comment (haslinghuis) [inline configs/TUNERCF435/config.h:64]: Please use spaces instead of tabs

Comment (TUNERC-Aria) [reply configs/TUNERCF435/config.h]: The config file is modified.
Please check it out.
Thank you.

File: configs/TUNERCF435/config.h [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_AUTO
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define BEEPER_INVERTED
+#define SYSTEM_HSE_MHZ          8
+#define GYRO_1_SPI_INSTANCE     SPI1
+#define MAX7456_SPI_INSTANCE    SPI2
+#define FLASH_SPI_INSTANCE      SPI3
+#define BARO_I2C_INSTANCE       (I2CDEV_2)
```
Comment (ot0tot) [inline configs/TUNERCF435/config.h:106]:
```suggestion
#define BARO_I2C_INSTANCE       (I2CDEV_2)
#define MAG_I2C_INSTANCE        (I2CDEV_2)
```

File: configs/TUNERCF435/config.h [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                                TIMER_PIN_MAP(4, PB6, 1, 0)
+
+#define ADC_VBAT_PIN            PB0
+#define ADC_CURR_PIN            PB1
+#define ADC_INSTANCE            ADC1
+#define ADC1_DMA_OPT            11
+
+
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_AUTO
```
Comment (ot0tot) [inline configs/TUNERCF435/config.h:98]: Bitbang should be enabled by default.
```suggestion
#define DEFAULT_DSHOT_BURST               DSHOT_DMAR_AUTO
#define DEFAULT_DSHOT_BITBANG           DSHOT_BITBANG_ON
```

**Comment by ot0tot** [comment]
Minor nitpick, but please use spaces to align all of the define parameters. Looks good otherwise 👍

**Pull request merged**